docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -1,39 +1,178 @@
+| [`handlePropertiesPolymorphically`](#attribute-handlepropertiespolymorphically) | No        | `boolean`                                                                                                                    | `false` |
+| [`instanceFilter`](#attribute-instancefilter)                                   | No        | [ECExpression](./ECExpressions.md#instance-filter)                                                                           | `""`    |
+| [`onlyIfNotHandled`](#attribute-onlyifnothandled)                               | No        | boolean                                                                                                                      | `false` |
+| *Ordering*                                                                      |
+| [`priority`](#attribute-priority)                                               | No        | `number`                                                                                                                     | `1000`  |
+| *Content Modifiers*                                                             |
+| [`relatedProperties`](#attribute-relatedproperties)                             | No        | `RelatedPropertiesSpecification[]`                                                                                           | `[]`    |
+| [`calculatedProperties`](#attribute-calculatedproperties)                       | No        | `CalculatedPropertiesSpecification[]`                                                                                        | `[]`    |
+| [`propertyCategories`](#attribute-propertycategories)                           | No        | `PropertyCategorySpecification[]`                                                                                            | `[]`    |
+| [`propertyOverrides`](#attribute-propertyoverrides)                             | No        | `PropertySpecification[]`                                                                                                    | `[]`    |
+| [`showImages`](#attribute-showimages)                                           | No        | `boolean`                                                                                                                    | `false` |
+| *Misc.*                                                                         |
+| [`relatedInstances`](#attribute-relatedinstances)                               | No        | [`RelatedInstanceSpecification[]`](../Common-Rules/RelatedInstanceSpecification.md)                                          | `[]`    |
+
+### Attribute: `classes`
+
+Defines a single or an array of [multi schema classes](../Common-Rules/MultiSchemaClassesSpecification.md) which specify what and how the classes need to be selected to form result content.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.Classes.Ruleset]]
```

Comment:
The instances are from Model class which, as far as I've understood, don't have many properties since those are located in the elements that the model represents.

Reply:
Correct, the base `Model` class has only 1 visible property.